docs/user-guide/tftp.rst

```diff
@@ -0,0 +1,90 @@
+******************
+The TFTP Directory
+******************
+
+For booting machines in a PXE and/or HTTP-Boot environment the TFTP directory is the most important directory. This
+folder contains all static files required for booting a system.
+
+The folder of this is dependant on your distro and can be changed in the Cobbler settings. The default should be
+correctly set during the package build of your Linux distro or during the installation process (if you are use the
+source installation).
+
+Behaviour
+#########
+
+A good explanation of ``cobbler sync`` can be found here: :ref:`cli-cobbler-sync`
+
+In the following we will examine the behaviour for the TFTP directory more in details.
+
+#. ``cobbler sync`` is executed (we assume a full one for now).
+#. The pre-sync triggers are executed.
+#. If the following directories do not exist they are created:
+    #. ``pxelinux.cfg``
+    #. ``grub``
+    #. ``images``
+    #. ``ipxe``
+    #. A symlink from ``grub/images`` to ``images``
+#. The content of in above mentioned directories is being fully deleted.
+#. All bootloaders are being copied
+#. All kernel and initrds are being copied
+#. All images (if created) are being copied
+#. The PXE menu is being generated and written to disk
+#. The post-sync triggers are being executed
+
+.. note:: If you only sync DHCP, DNS or specific systems the order and actions might be slightly different.
+
+.. warning:: A ``cobbler sync`` is not required. Due to the file copying of a lot of small files this is a very
```

Comment:
Just as a reminder: if possible, try to avoid consecutive admonitions. It make them less significant. But for the time being, leave it as.

Reply:
Thank your for the suggestion. Will try to avoid this in the future.